src/invite/InviteModal.tsx

```diff
@@ -77,10 +111,28 @@ const styles = StyleSheet.create({
     width: 120,
     height: 120,
   },
+  helpContainer: {
+    width: 312,
+    height: 100,
+    left: 24,
+  },
+  helpText: {
+    color: colors.gray5,
+    fontSize: 12,
+    textAlign: 'center',
+  },
+  helpLink: {
+    color: colors.onboardingBlue,
+    flexWrap: 'wrap',
+    textDecorationLine: 'underline',
+  },
   text: {
     textAlign: 'center',
     marginBottom: Spacing.Regular16,
   },
+  textBold: {
+    fontFamily: 'Inter-SemiBold',
```

Comment:
would use `...fontStyles.xsmall, fontFamily: 'Inter-SemiBold'` to control the font size etc